tests/core/snapshots/test-multiple-resolvable-directives-on-field.md_error.snap

```diff
@@ -4,19 +4,8 @@ expression: errors
 ---
 [
   {
-    "message": "Multiple resolvers detected [@http, @expr]",
-    "trace": [
-      "Query",
-      "user1"
-    ],
-    "description": null
-  },
-  {
-    "message": "Multiple resolvers detected [@http, @call]",
-    "trace": [
-      "Query",
-      "user2"
-    ],
+    "message": "Validation Error\n• Multiple resolvers detected [@http, @expr] [Query, user1]\n• Multiple resolvers detected [@http, @call] [Query, user2]\n",
```

Comment:
The error structure that separated the message from trace and description is lost over here.

Reply:
fixed